.prettierignore

```diff
@@ -0,0 +1,2 @@
+cache
```

Comment:
In the other Octokit repos, we specify the paths to prettier itself that we want linted. It avoids having extra configuration files

Reply:
✅ Thanks for the feedback! Fixed.